src/main/java/io/mosip/mimoto/util/Utilities.java

```diff
@@ -103,6 +103,12 @@ public class Utilities {
     @Value("${mosip.openid.issuers}")
     private String getIssuersConfigJson;
 
+    @Value("${mosip.openid.issuer.credentialSupported}")
```

Comment:
shouldn't this wellknown be fetched from issuers config based on the issuer ?

Reply:
we have the code to fetch it from well known endpoint from issuer config. this is a double check in case well known endpoint returns null